comprobo_final_project/scripts/chromosome.py

```diff
@@ -90,8 +92,13 @@ def get_fitness(self):
         """
 
         self.supervisor.use_genes(self.genes)
-        xpos, ypos = self.supervisor.run()
-        fitness = np.sqrt(xpos**2 + ypos**2)
+
+        xpos1, ypos1, xpos2, ypos2, xpos3, ypos3 = self.supervisor.run()
+        x = [xpos1, xpos2, xpos3]
+        y = [ypos1, ypos2, ypos3]
+
+        slope, intercept, r_value, p_value, std_err = stats.linregress(x,y)
+        fitness = r_value**2
         self.supervisor.reset()
         return fitness
```

Comment:
It would be great to be able to modularly swap out which fitness functions we want to use. (Choosing between going to a point, making a line, playing tag, etc.)